tools.go

```diff
@@ -7,4 +7,5 @@ package tools
 
 import (
 	_ "github.com/go-swagger/go-swagger/cmd/swagger"
+	_ "gitea.com/jolheiser/gitea-vet"
```

Comment:
We can move this to `build/vendor.go`

Reply:
It actually can't go in that file because `gitea.com/jolheiser/gitea-vet` is a binary rather than a library, so an `_` import would fail.

Reply:
Why not a sub-repo?